### DISCUSSION TITLE
[8/24/2023] New Mojo Release!

### DISCUSSION THREAD

**Discussion by goldiegadde** [modular/modular#520] [Announcements]
Hey everyone 👋 ,

We released a new version of Mojo🔥 to Playground yesterday! To see what's changed, [check out the changelog](https://docs.modular.com/mojo/changelog.html).

If you have an active session in the Mojo Playground, you'll need to restart it to get the updated build.

Thanks,
Goldie

**Comment by drunkwcodes** [comment]
I LOVE this change!

**Comment by martin-braun** [comment]
I want to use Mojo for CLI tooling, I have no problem with breaking changes. When can we try it without the Playground, please?

**Reply by lattner** [reply]
We've announced that Mojo will be generally available for download in early september, so a few weeks.

**Reply by martin-braun** [reply]
@lattner That's fantastic news. I couldn't find that information and I'm positively surprised. Obviously, I will take it with a grain of salt, but it's such great news that are talking about weeks instead of months or years. Highly appreciated. 🙏

**Reply by lattner** [reply]
I guess it's a salty Mojo on the way!
